sklearn/tree/tests/test_export.py

```diff
@@ -21,6 +22,9 @@
 w = [1, 1, 1, 0.5, 0.5, 0.5]
 y_degraded = [1, 1, 1, 1, 1, 1]
 
+# constructors for feature names and class names
+constructors = [list, np.array]
```

Comment:
probably better to use this as a `pytest.mark.parametrize` on the test instead.

Reply:
@adrinjalali Thanks for your suggestion! I actually considered that, however the test `test_graphviz_toy` has many toy tests inside, but not all of them need that constructor. I'm afraid that using `pytest.mark.parametrize` will cause some of the toy tests to be run multiple times.

Reply:
So it means that we need to decouple the check and have 2 tests. Alternatively, if this test is super fast, we can run it twice (that is the number of constructors).

Reply:
yeah, seems like we can probably split the test into more tests.